pyproject.toml

```diff
@@ -45,6 +45,7 @@ packages = ["ocrd_pagetopdf", "ocrd_pagetopdf.lib", "ocrd_pagetopdf.data"]
 
 [tool.setuptools.dynamic]
 dependencies = {file = ["requirements.txt"]}
+optional-dependencies.test = {file = ["requirements-test.txt"]}
```

Comment:
Is there any practical consequence to not adding the `requirement-test.txt` to `pyproject.toml`? Because I never do 😬

Reply:
IIRC some installers (poetry?) can run tests themselves (i.e. without our makefile), but I am not sure whether and how this is standardised...